mqt/syrec/syrec_editor.py

```diff
@@ -272,9 +272,9 @@ def build(self):
         self.circ = syrec.circuit()
 
         if self.synthesize_with_additional_lines:
```

Comment:
these variables still use the old naming and need to be renamed

Reply:
Done